apps/carbon/src/App.tsx

```diff
@@ -36,7 +36,7 @@ const extensions = [
 const App = (props: AppProps) => (
   <ErrorBoundary>
     <AppShellProvider theme={`${props.theme ? props.theme : "theme-dark"}`}>
-      <AppShell embedded>
+      <AppShell embedded fullWidthContent={undefined} sideNavigation={undefined} topNavigation={undefined}>
```

Comment:
do we really need to supply undefined for optional props?

Reply:
No, we do not but during the migration it was a mistake, so now we have a lot of left overs that we need to fix. This is a temporary solution. Plan is to move on and fix all leftovers

Reply:
agree, let's move on.